fastestimator/estimator.py

```diff
@@ -61,7 +63,7 @@ def __init__(self,
                  network: BaseNetwork,
                  epochs: int,
                  max_steps_per_epoch: Optional[int] = None,
-                 traces: Union[None, Trace, Iterable[Trace]] = None,
+                 traces: Union[None, Trace, Iterable[Union[Trace, Scheduler[Trace], Scheduler[Trace]]]] = None,
```

Comment:
why is scheduler trace inside the union twice?

Reply:
fixed